NEWS.md

```diff
@@ -37,3 +37,9 @@
 * Replace `snakecase::to_snake_case()` with `janitor::clean_names()` and make sure variable names specified in `keep` and `rename` are preserved after applying `janitor::clean_names()` (#180, Karim-Mane).
 
 * Function documentations have been updated. The `clean_data()` no longer takes the `params` argument. Instead, it takes many other arguments that are described in the function documentation and the package vignettes (#184, Karim-Mane).
+
+* Use {cli} package for formatting messages in the package (#196, Karim-Mane).
+
+* Translate messages in the package into French using {potools} package (#196, Karim-Mane).
+
+* An additional column (**row_id**) with the row indices where the incorrect date sequences are found is now added to the report made from the `check_date_sequence()` function (#196, Karim-Mane).
```

Comment:
Ideally, each one of these points should be a single PR to simplify review

Reply:
I agree with you. I thought about it several times before deciding to proceed this way. Reformatting the messages with {cli} and translating them into French with {potools} are targetting the same parts of the code. This was the main reason why I combined both in the same PR.

I must admit that the third point on the other side was not inline with the above and would have been in a separate PR. But I decided to solve this here to allow for the RFs to use the enhanced version of the function during the pre-summit training (which unfortunately did not happen).

Reply:
It's not a big deal now but for what it's worth, splitting the PR might have allowed the RFs to use the enhanced version since the other PR would have been much smaller and easier to review and therefore merged faster. Smaller PRs actually allow faster integration of new features.

Admittedly, there is a small overhead for the maintainer since you have to maintain multiple branch (and rebase them when necessary) and it requires a strong discipline to ensure you keep the scope but I believe the benefits (cleaner history, faster releases, easier reviews, etc.) are really worth it.

Reply:
Thanks for this good reminder about the benefit of small PRs. I was already doing so, but will reinforce myself toward achieving this.